shell/platform/android/test/io/flutter/embedding/android/FlutterActivityTest.java

```diff
@@ -112,6 +113,26 @@ public void itCreatesNewEngineIntentWithRequestedSettings() {
     assertEquals(TransparencyMode.transparent, flutterActivity.getTransparencyMode());
   }
 
+  @Test
+  public void itCreatesNewEngineIntentWithLaunchUri() {
+    Intent intent = FlutterActivity.createDefaultIntent(RuntimeEnvironment.application);
+    intent.setData(Uri.parse("http://myApp/custom/route"));
```

Comment:
Hey @chunhtai what is up with the `myApp` portion of the URL?  It seems to be ignored.

Reply:
yes, currently it only returns URI.path.
I will change to return full URI per our discussion.